README.md

```diff
@@ -705,6 +705,7 @@ These plugins add tab completions without adding extra functions or aliases.
 * [umake](https://github.com/zlsun/umake) - Tab completion for Ubuntu umake.
 * [vert.x-omz-plugin](https://github.com/davidafsilva/vert.x-omz-plugin) - Provides autocomplete features for the [vertx](https://vertx.io/) command.
 * [yarn](https://github.com/g-plane/zsh-yarn-autocompletions) - Yarn autocompletions.
+* [zsh-antibody-completion](https://github.com/sinetoami/zsh-antibody-completion) - This plugin provides completion for [Antibody](https://github.com/getantibody/antibody) plugin manager
```

Comment:
Please trim off the leading `zsh-` as shown below - that keeps the majority of entries from clustering in the **Z** part of the list.

```suggestion
* [antibody-completion](https://github.com/sinetoami/zsh-antibody-completion) - This plugin provides completion for the [Antibody](https://github.com/getantibody/antibody) plugin manager
```

Reply:
E really appreciate your tip! Thanks :)